assets/queries/openAPI/api_key_exposed_in_global_security/query.rego

```diff
@@ -0,0 +1,20 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) != "undefined"
+
+	security := doc.security[x][s]
+	ins := {"cookie", "header", "query"}
+	doc.components.securitySchemes[s].in == ins[z]
```

Comment:
Since the problem is using `apiKey` type, I think we could check `doc.components.securitySchemes[s].type == "apiKey`, no?